downloader/src/main/kotlin/vcs/GitRepo.kt

```diff
@@ -151,7 +151,8 @@ class GitRepo : VersionControlSystem(), CommandLineTool {
             "Initializing git-repo from ${vcs.url} with revision '$manifestRevision' and manifest '$manifestPath'."
         }
 
-        // Clone all projects instead of only those in the "default" group until we support specifying groups.
+        // Prepare for cloning all projects instead of only those in the "default" group until we support specifying
+        // groups.
```

Comment:
Would it make sense to re-format the below code to one param per line and then move the comment next to `--groups=all` as it only refers to that option?